src/Lykke.Service.ChainalysisProxy/Startup.cs

```diff
@@ -75,7 +77,13 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IApplica
         {
             try
             {
-                if (env.IsDevelopment())
+                Mapper.Initialize(cfg =>
```

Comment:
Do not use static mapper it can conflicts with other nugets which can also have they own static registration. Instance is safer.

Reply:
There is no conflict for now. Will change if will. No other changes for the proxy awaiting.

Reply:
But it can happen in a future. As addition advantage you can always control mapping in tests.